tests/metal_problem_regression.in

```diff
@@ -0,0 +1,38 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  0.0       0.0       -2.4144e+22
+geometry.prob_hi     =  3.018e21  3.018e21   2.4144e+22
+geometry.is_periodic =  1    1    0
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 1      # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 32 32 256
+amr.max_level       = 0     # number of levels = max_level + 1
+amr.max_grid_size   = 128   # at least 128 for GPUs
+amr.blocking_factor = 32   # grid size must be divisible by this
+amr.n_error_buf     = 3     # minimum 3 cell buffer around tagged cells
+amr.grid_eff        = 0.7   # default
+
+do_reflux     = 1
+do_subcycle   = 1
+stop_time     = 9.462e+15
+max_timesteps = 100
```

Comment:
It could run for a few thousand steps here. I don't think it needs to be as short as 100 steps.